src/components/Rockets.js

```diff
@@ -1,8 +1,46 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
- [ ] Kindly make the `Rockets` page the default view as requested by the previous reviewer by changing the defined route for the Rockets component to `/`  instead of `/Rockets`. so you can access the Rockets page with the default route `/`. Also don't forget to update the Navlink related to the Rockets page in the Navigation component to the new route.